app/controllers/course_summaries_controller.rb

```diff
@@ -45,7 +45,8 @@ def generate_csv_header(csv)
     grade_entry_forms = GradeEntryForm.all
     marking_schemes = MarkingScheme.all
 
-    header = ['Username']
+    header = [I18n.t('grade_entry_forms.username')]
+    header.concat([I18n.t('grade_entry_forms.id_number')])
```

Comment:
It's unnecessary to use two separate lines; just initialize the `header` to be a list containing both strings. Similar to the test down below.